### PR TITLE
Added Wattpad

### PR DESCRIPTION
**Pull request by xhendrikg**

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -9993,6 +9993,16 @@
...
+    {
+        "name": "Wattpad",
+        "url": "https://www.wattpad.com/settings",
+        "difficulty": "easy",
+        "notes": "Login > Close Account",
+        "domains": [
+            "wattpad.com"
+        ]
+    },
+
```
Comment (tupaschoal) [inline _data/sites.json:10005]:
```suggestion

```

**Review by tupaschoal — COMMENTED** [review]
Do you happen to know whether close account actually deletes the data? (They typically say so)

**Comment by xhendrikg** [comment]
> Do you happen to know whether close account actually deletes the data? (They typically say so)

Sorry i don't know but i think they dont because I searched for that and could get an answer. With search I mean search on the page where you can delte the account.

File: _data/sites.json [hunk 2]
```diff
@@ -9993,6 +9993,15 @@
         ]
     },
 
+    {
+        "name": "Wattpad",
+        "url": "https://www.wattpad.com/settings",
+        "difficulty": "easy",
+        "notes": "Login > Close Account",
```
Comment (tupaschoal) [inline _data/sites.json:10000]:
```suggestion
        "notes": "You can close your account by going to Login > Close Account. Note that this doesn't delete it nor anything related to it and trying to log back in reactivates it instantly",
```

**Pull request merged**